CONTRIBUTING.md

```diff
@@ -45,6 +45,11 @@ What constitutes a business day is managed by the gem `business_time` and is con
 
 ### App setup
 
+Before running bin/setup, ensure that 'foreman' is removed from the Gemfile.  
```

Comment:
oh yeah...that is totally my fault. Usually I use foreman locally only so I do not include it in the Gemfile. This is because it's usually a best practice to leave development dependencies that are not required out of the project. eg: I might like foreman but maybe you just want to use the regular rails server.

In this project, however, we are using foreman in all envs so we need it. Let's merge this as-is and I will open a separate Pr to deal with foreman instructions.